scalajvm/app/views/main.scala.html

```diff
@@ -24,6 +24,6 @@
             <li class="menu-service"><a href="@routes.SuggestedQuotes.newQuote()">Добавить цитату</a></li>
 	    </ul>
         <div class="content center-content">@content</div>
-        <div class="footer center-content"><span class="highlight"><a href="https://github.com/codingteam/loglist">codingteam</a> ©</span> 2015</div>
+        <div class="footer center-content"><span class="highlight"><a href="https://github.com/codingteam/loglist">codingteam</a> ©</span> 2016</div>
```

Comment:
Also [StackOverflow](http://stackoverflow.com/) doesn't seem to use year range. That's a nice example.